modules/olm-accessing-images-private-registries.adoc

```diff
@@ -22,10 +22,11 @@ Instead, the authentication details can be added to the global cluster pull secr
 
 .Prerequisites
 
-* At least one of the following hosted in a private registry:
+* You have at least one of the following hosted in a private registry:
 ** An index image or catalog image.
 ** An Operator bundle image.
 ** An Operator or Operand image.
+* You have access to the cluster as a user with the `cluster-admin` role.
```

Comment:
dedicated-admin for ROSA/OSD?

Reply:
This procedure is excluded from OSD and ROSA in the assembly in which it is used.